unity/Assets/Scripts/ArticulatedAgentSolver.cs

```diff
@@ -129,7 +133,13 @@ public class ArticulatedAgentSolver : MonoBehaviour, MovableContinuous {
                 float relativeForce = (speedDelta / Time.fixedDeltaTime) * currentAgentMoveParams.agentMass
                     * currentAgentMoveParams.direction;
 
-                Debug.Log("3. speedDelta is " + speedDelta + ". Applying force of " + relativeForce);
+                relativeForce = Mathf.Clamp(relativeForce, -currentAgentMoveParams.maxForce, currentAgentMoveParams.maxForce);
+                Debug.Log(
+                    $"3. speedDelta is {speedDelta}. Applying force of {relativeForce} = "
+                    + $"clamp({currentAgentMoveParams.maxForce}, {speedDelta} / {Time.fixedDeltaTime}) * {currentAgentMoveParams.agentMass} * {currentAgentMoveParams.direction})."
```

Comment:
Jus for debug but use `fixedDeltaTime` instead

Reply:
Done